.github/workflows/integration-test.yml

```diff
@@ -435,12 +311,11 @@ jobs:
       - test-integration-loki
       - test-integration-pulsar
       - test-integration-splunk
-      - check-component-features
     runs-on: ubuntu-latest
     steps:
     - name: Discord notification
       env:
         DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
       uses: Ilshidur/action-discord@0.3.0
       with:
-        args: "Master failed and is red: <https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}>"
+        args: "Master integration-test.yml failed: <https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}>"
```

Comment:
Should this be `Master Integration Tests failed..."

Reply:
:+1: that's fair :smile: